### ISSUE TITLE
Publish solutions to the exercises in appendix?

### ISSUE THREAD

**Issue by ilante** [walkccc/CLRS#424]
Thank you so much for sharing your effort of publishing the solutions to this book!

I was wondering if anyone in the community has solved the exercises from the appendix and wants to add a pull request for that?

**Comment by jxu** [comment]
I agree, although most are basic enough to find online, not specific to this book.
